collect_app/src/main/java/org/odk/collect/android/activities/viewmodels/MainMenuViewModel.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.activities.viewmodels;
+
+import androidx.annotation.Nullable;
+import androidx.lifecycle.ViewModel;
+
+import org.odk.collect.android.BuildConfig;
+
+public class MainMenuViewModel extends ViewModel {
+
+    public String getVersion() {
+        return versionNameWithoutPrefix().split("-")[0];
+    }
+
+    @Nullable
+    public String getVersionSHA() {
+        String[] split = versionNameWithoutPrefix().split("-", 2);
```

Comment:
This means that for beta versions the beta will be at the bottom of the screen. I think of the beta as being part of the version (e.g. we put v1.26.0-beta.0 in the release notes) and I think it's helpful to have up top.

Reply:
Make sense. I'll change it!